spec/spec/requests/posts_spec.rb

```diff
@@ -2,6 +2,26 @@
 
 RSpec.describe 'Posts', type: :request do
```

Comment:
- [ ] All your tests are failing. For one, I think it has to do with your Posts controller. You are inheriting PostsController from a non-existent ApplicationController. Kindly fix these issues and fix all failing tests.

  ![idowp](https://user-images.githubusercontent.com/54848363/219060526-dabc4f52-9165-4bd2-8d5f-2b2d1e1d1da0.jpg)
  ![oew](https://user-images.githubusercontent.com/54848363/219060539-e26d33dc-3fc3-466c-9a97-3c36fffb187f.jpg)